src/index.js

```diff
@@ -12,7 +12,7 @@ export default function loader(content) {
 
   const context = options.context || this.rootContext || (this.options && this.options.context);
 
-  let url = loaderUtils.interpolateName(this, options.name, {
+  const originURL = loaderUtils.interpolateName(this, options.name, {
```

Comment:
`originURL` => `name || file`

Reply:
@michael-ciniawsky agree, but `interpolateName` return `url` (https://github.com/webpack/loader-utils/blob/master/lib/interpolateName.js#L96), maybe better leave as is (`originURL` || `originalURL`)